### PR TITLE
[DB] Use Postgre as database

### PR DESCRIPTION
**Pull request by hansenedrickh**
This PR is for adding support of postgre as main database to avoid data loss

### CODE REVIEW

File: dictionary/dictionary.go [hunk 1]
```diff
@@ -10,9 +10,9 @@ var MemberRemoveMessages = []string{
 	"removed %s from `%s`",
 }
 
-// MemberGetMessages is the list of message which can be returned when triggering get command
-var MemberGetMessages = []string{
-	"amazing members of `%s` are %s",
+// MemberCheckMessage is the list of message which can be returned when triggering get command
```
Comment (fwilson7) [inline dictionary/dictionary.go:13]:
```suggestion
// MemberCheckMessage is the list of message which can be returned when triggering check command
```

File: helper/helper.go [hunk 2]
```diff
@@ -21,8 +22,26 @@ func GetCommand(msg string) string {
...
+func ReplaceQueryString(s string) string {
+	replacer := strings.NewReplacer(`(`, `\(`,
+		`)`, `\)`,
+		`$`, `\$`)
+
+	return replacer.Replace(s)
+}
+
+// TransformSliceInterfaceToSliceValue is the helper for making an argument for DB query
+func TransformSliceInterfaceToSliceValue(i []interface{}) (res []driver.Value) {
```
Comment (fwilson7) [inline helper/helper.go:41]: looks like not used?

File: helper/helper.go [hunk 3]
```diff
@@ -21,8 +22,26 @@ func GetCommand(msg string) string {
...
-// GetRandomMessage is the helper for getting one random message from message list
-func GetRandomMessage(messages []string) string {
+// GetRandom is the helper for getting one random message from message list
+func GetRandom(messages []string) string {
 	result := rand.Intn(len(messages))
 	return messages[result]
 }
+
+// ReplaceQueryString is the helper for making a query string
+func ReplaceQueryString(s string) string {
```
[the remark this answers is not in the document]
Comment (hansenedrickh) [reply helper/helper.go:32]: Used in the test for creating sqlmock

File: helper/helper.go (hunk 2 above)
Comment (hansenedrickh) [reply helper/helper.go:41]: Used in the test for creating sqlmock

**Pull request merged**